nativescript-angular/animations/animation-player.ts

```diff
@@ -41,7 +41,9 @@ export class NativeScriptAnimationPlayer implements AnimationPlayer {
     onDestroy(fn: Function): void { this._doneSubscriptions.push(fn); }
 
     play(): void {
-        traceLog(`NativeScriptAnimationPlayer.play`);
+        if (isLogEnabled()) {
+            traceLog(`NativeScriptAnimationPlayer.play`);
+        }
```

Comment:
@sis0k0 just curious - couldn't this perhaps just be called in tracer itself to avoid having to have these conditionals all over the codebase? ie:
```
import { isEnabled } from "tns-core-modules/trace";

export function animationsLog(message: string): void {
    if (isEnabled()) {
      write(message, animationsTraceCategory);
   }
}
```

Reply:
The disadvantage of having the check inside the tracer itself is that this will result in two function calls - one for `animationsLog` and one for `isEnabled`.

Reply:
Also if the check was inside the `animationsLog` function and we invoked `animationsLog("something" + random)', we would have the overhead of the string concatenation as well.

Reply:
Thanks for explanation makes sense :) Perhaps there’s a possibility we could also add something to webpack config to strip these all out in production mode as well to coincide with this pr. Ideally with the whole trace enabler control we should probably have ways that devs can strip them when desired as well since keeping core libs slim and performant as possible is always highest goal (which thank you for attention to that here!).

Reply:
That's a great idea! 10x :)